test/fixtures/dns_forward_and_reverse/example.tf

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
I'm all for this and will definitely make the change, but the reason why I didn't do it that way was because the Kubernetes module has been positioned as the example, and it currently uses `example.tf` https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/tree/master/test/fixtures/deploy_service 

Absolutely still in favor of `main.tf`, but bringing up that we should probably target this change for the Kubernetes module too